graphql/database/users/__tests__/getLandingPagePhrase.test.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env jest */
+
+import { V4_SUPPORTING_STATEMENTS } from '../../experiments/experimentConstants'
+
+import { getMockUserContext, getMockCauseInstance } from '../../test-utils'
+import getUserFeature from '../../experiments/getUserFeature'
+import Feature from '../../experiments/FeatureModel'
+
+jest.mock('../../experiments/getUserFeature')
+
+const cause = getMockCauseInstance()
+const userContext = getMockUserContext()
+
+describe('getLandingPagePhrase tests', () => {
+  it('returns constant phrase', async () => {
+    expect.assertions(1)
+    const getLandingPagePhrase = require('../getLandingPagePhrase').default
+
+    getUserFeature.mockResolvedValueOnce(
+      new Feature({
+        featureName: V4_SUPPORTING_STATEMENTS,
+        variation: 'Constant',
```

Comment:
Nitpick: this variation should be "Control". No effect on the test behavior, though.

Reply:
Good catch. Fixed.